deploy/crds/Keycloak.yaml

```diff
@@ -29,6 +29,28 @@ spec:
           type: object
         spec:
           type: object
+          properties:
+            externalDatabaseSecret:
+              description: 'Opotional. A Secret containing the connection string and credentials to an external database. If not provided a Postgres
```

Comment:
```suggestion
              description: 'Optional. A Secret containing the connection string and credentials to an external database. If not provided a Postgres
```

Reply:
Should this description also list the properties required in the secret or a link to where they can get that information?

Reply:
Yep, good point.